src/utils/trust.details.ts

```diff
@@ -62,7 +63,8 @@ const getPageProperties = async (
 
   return {
     backLinkUrl: getBackLinkUrl(isUpdate, appData, req, isReview),
-    templateName: getPageTemplate(isUpdate, isReview),
+    templateName: getPageTemplate(isUpdate, isReview, req.url).templateName,
```

Comment:
I would call getPageTemplate(isUpdate, isReview, req.url) once before the return to get both the templateName and template values.